api/v1/views/index.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+"""index of views"""
+from api.v1.views import app_views
+from flask import jsonify
+
+@app_views.route('/status', strict_slashes=False)
+def status():
+    """status of api v1"""
+    status = {"status": "OK"}
```

Comment:
I share your curiosity, mentioned in your intro comments to the PR for `Task6 v2` branch, about the newlines in the example output, eg:
{
  "status": "OK"
}
 Still researching, maybe we can fix later if needed. Seems like since the object passed is a dict and not a string, then any whitespace in its declaration wouldn't matter. An extra kwarg to `jsonify`, or something on the other end governing the display, I wonder?